tests/introspection/ApiSignatureTest.cs

```diff
@@ -978,5 +984,13 @@ protected virtual bool IgnoreAsync (MethodInfo m)
 			}
 			return false;
 		}
+
+		protected HashSet<string> BindAsSupportedTypes = new HashSet<string> {
```

Comment:
Love it. Worth having a bidi comment with the generator source to keep this in sync